go/ql/test/library-tests/semmle/go/security/FileSystemAccess/main.go

```diff
@@ -0,0 +1,136 @@
+package main
+
```

Comment:
Please include the go generate commands for depstubber here, in case the stubs need to be regenerated in future. [Here](https://github.com/github/codeql/blob/60b7840389f770b9fb0e923ac1b153989cdbb551/go/ql/test/library-tests/semmle/go/frameworks/NoSQL/main.go#L3-L6) is an example.